src/tpm2_wrap.c

```diff
@@ -1052,23 +1044,20 @@ int wolfTPM2_EncryptSalt(WOLFTPM2_DEV* dev, WOLFTPM2_KEY* tpmKey,
     #endif
     #if !defined(NO_RSA) && !defined(WC_NO_RNG)
         case TPM_ALG_RSA:
-            rc = wolfTPM2_RSA_Salt(dev, tpmKey, salt, &in->encryptedSalt,
-                &tpmKey->pub.publicArea);
+            rc = wolfTPM2_EncryptSalt_RSA(dev, tpmKey, salt, encSalt, label);
             break;
     #endif
         default:
             rc = NOT_COMPILED_IN;
             break;
     }
-#else
-    (void)dev;
-    (void)in;
-    (void)salt;
     rc = NOT_COMPILED_IN;
```

Comment:
`wolfTPM2_EncryptSalt` is always returning `NOT_COMPILED_IN`, I think this `rc = NOT_COMPILED_IN` is from the old code and you just missed deleting this line

Reply:
actually now that I look at it why was that else macro removed? if `WOLFTPM2_NO_WOLFCRYPT` is defined rc will be undefined before return, you could probably change it to `int rc = NOT_COMPILED_IN;` at the start of the function